modules/frontend/handler.go

```diff
@@ -81,35 +86,40 @@ func (f *handler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		f.post(ctx, resp, orgID, elapsed, err)
 	}
 
+	logMessage := append([]interface{}{
+		"tenant", orgID,
+		"method", r.Method,
+		"traceID", traceID,
+		"url", r.URL.RequestURI(),
+		"duration", elapsed.String(),
+	})
```

Comment:
This should get rid of the lint error

```suggestion
	logMessage := []interface{}{
		"tenant", orgID,
		"method", r.Method,
		"traceID", traceID,
		"url", r.URL.RequestURI(),
		"duration", elapsed.String(),
	}
```